internal/website/static/css/style.css

```diff
@@ -126,6 +125,39 @@ body {
 .Sidenav-pageLink:active {
   text-decoration: underline;
 }
+.PageTOC {
+  border-left: 2px solid #dbd9d6;
+  float: right;
+  margin: 0.25rem 0 1rem 2rem;
+  max-width: 25%;
+  min-width: 15%;
+  padding: 0.5rem 0 0.5rem 1rem;
+}
+@media (max-width: 1024px) {
+  .PageTOC {
+    border: none;
+    float: none;
+    margin: 1.5rem 0 0.5rem;
+    max-width: none;
+    min-width: none;
+    padding: 0;
+  }
+}
+.PageTOC-heading {
+  font: 700 1rem/1.25 'Work Sans', 'Roboto', sans-serif;
```

Comment:
WDYT about making the font smaller? Maybe everywhere (not just TOC), but that's a separate question :-). At least in the screenshot you gave, the TOC is too long to fit on a page and wraps strangely, the font looks too big for me.

Reply:
At any legible font size, the page TOC is going to spill off the page for any of our larger (read as: opening) how-to guides, especially on smaller screens. I put in a fair amount of care to make sure the layout doesn't break when this happens.

If you want the font smaller, I'd like to do that as a separate global change. I think some of the issue is coming from the fact that the top header pushes content below the fold.

Reply:
Opened #2504 for the font size change.

Reply:
+ 1 to size change for the content. Would TOC size need to be a separate change as well? Seems like it is visually independent of content and can be done here. Also, I think it is important to preserve the nesting structure of the headers.

Reply:
Done.